training.py

```diff
@@ -55,6 +55,7 @@ def main(args):
         pickle.dump(args, f)
 
     if args.use_traj_idx_wrapper:
+        raise Exception("traj_index_wrapper_factory does not yet support the new goal indexing scheme.")
```

Comment:
In the future we will probably want remove trajectory index wrapper, it was made only for debugging of replay buffer, and now it only makes code complicated.